components/Entry/index.js

```diff
@@ -1,15 +1,29 @@
 import styled from "styled-components";
 import { StyledCard } from "../StyledComponents/StyledCard";
+import { StyledSubmitButton } from "../StyledComponents/StyledSubmitButton";
+
+function Entry({ entry, handleAddEntry }) {
+  const { japanese, english, showAddButton, isDictionaryEntry } = entry;
 
-function Entry({ JPDefinition, JPReading, ENDefinition }) {
   return (
     <StyledCard>
-      <StyledJPDefinition>{JPDefinition}</StyledJPDefinition>
+      {showAddButton ? (
+        <StyledSubmitButton type="button" onClick={() => handleAddEntry(entry)}>
+          +
+        </StyledSubmitButton>
+      ) : !showAddButton && isDictionaryEntry ? (
+        <StyledSubmitButton type="button" disabled="true">
+          ✔
+        </StyledSubmitButton>
+      ) : (
+        ""
+      )}
```

Comment:
Instead of using a chained ternary operator, I would suggest going with the logical and. This would make this section of your code a lot easier to read:

```suggestion
       {showAddButton && (
        <StyledSubmitButton type="button" onClick={() => handleAddEntry(entry)}>
          +
        </StyledSubmitButton>
      )}
      {!showAddButton && isDictionaryEntry && (
        <StyledSubmitButton type="button" disabled="true">
          ✔
        </StyledSubmitButton>
      )}
```

Reply:
Yes! That looks more readable. Thank you. :)